modules/internal/ChapelArray.chpl

```diff
@@ -780,9 +780,9 @@ module ChapelArray {
       if __primitive("method call resolves", _value, "dsiNewRectangularDom",
                      rank, idxType, stridable, ranges2) {
 
-        //RSDW: compilerWarning("the domain map '", _value.type:string,
-        // " needs to be updated from stridable: bool to strides: strideKind",
-        // " because 'stridable' is deprecated");
+        compilerWarning("the domain map '", _value.type:string,
+          " needs to be updated from stridable: bool to strides: strideKind",
```

Comment:
can you put single quotes around `stridable: bool` and `strides: strideKind`, otherwise everything runs together